deptool.py

```diff
@@ -239,7 +241,13 @@ def do_whatprovides(self, subcmd, opts, *relation):
         for r in relation:
             i = self.pool.str2id(r)
             for s in self.pool.whatprovides(i):
-                print('- {}-{}@{}:'.format(s.name, s.evr, s.arch))
+                if opts.source:
+                    src = s.name
+                    if not s.lookup_void(solv.SOLVABLE_SOURCENAME):
+                        src = s.lookup_str(solv.SOLVABLE_SOURCENAME)
+                    print (src)
```

Comment:
please remove the whitespace after the function - no idea why flake doesn't hit you, this looks ugly :)

Reply:
any interest in getting this merged still?